### DISCUSSION TITLE
lineWithErrorBars

### DISCUSSION THREAD

**Discussion by mahadev-Pandhare** [sgratzl/chartjs-chart-error-bars#18] [Q&A]
Hi ,
We are getting issue with lineWithErrorBars option of plugin  with chartjs 3.6.2 verion.
So we are looking for sample how it works with line chart.

Looking for reply.

**Comment by sgratzl** [comment]
https://github.com/sgratzl/chartjs-chart-error-bars/blob/main/samples/lines.html should work without any problems
